hack/mirror-images.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if command -V skopeo; then
+    echo "skopeo is available"
+else
+    echo "skopeo is not available. installing..."
+    sudo dnf -y install skopeo
+fi
```

Comment:
No need for this if we use a recent bootstrap image

Reply:
Removed